x-pack/plugins/osquery/public/packs/queries/ecs_mapping_editor_field.tsx

```diff
@@ -394,7 +394,7 @@ const OsqueryColumnFieldComponent: React.FC<OsqueryColumnFieldProps> = ({
       >
         <EuiFlexItem grow={false}>
           <span css={fieldSpanCss} className="euiSuggestItem__label euiSuggestItem__label--expand">
-            {option.value.suggestion_label}
+            <b>{option.value.suggestion_label}</b>
```

Comment:
as we already have `fieldSpanCss` maybe we could add `font-weight: bold;` instead or `<b>`?